eng/packages/http-client-csharp-mgmt/generator/Azure.Generator.Management/src/Snippets/ResourceMethodSnippets.cs

```diff
@@ -120,7 +121,7 @@ public static IReadOnlyList<MethodBodyStatement> CreateGenericResponsePipelinePr
                 new CSharpType(typeof(Response<>), responseGenericType),
                 Static(typeof(Response)).Invoke(
                     nameof(Response.FromValue),
-                    [resultVariable.CastTo(responseGenericType), resultVariable]),
```

Comment:
LOL it is lucky for us that we fully control how to build the invocation of this - visitors cannot do this.